package/skelcd-control-openSUSE.spec

```diff
@@ -27,7 +27,7 @@
 #
 ######################################################################
 Name:           skelcd-control-openSUSE
-Version:        42.3.99.8
+Version:        42.3.99.9
```

Comment:
I think @ancorgs introduce specific versioning for storage-ng branch, so it won't fail when other changes to master is done. So better ask him